core/tests/test_suite.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Oppia-ml test suite.
```

Comment:
as I had raised the concern earlier - PRs should really be self-contained and should only contain directly relevant stuff. So, this PR should have only contained things that would directly help in "remote communication" and nothing else. I think that this code should have been a part of some other PR (something like - "setting up testing framework for Oppia-ml"). I strongly believe that PRs should be short and should try to tackle one problem at a time. In fact, this was the first advice that I got from Sean. [This](http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/) article perfectly summarizes why big PRs are hard to work with. @seanlip can you please give your advice? Thanks!